syft/serde/protobuf/torch_serde.py

```diff
@@ -5,6 +5,7 @@
 import io
 
 import torch
+import pydoc
```

Comment:
Haven't seen this used in the Syft code base before. 🤔

Reply:
Actually, I've used it in the typechecking PR when serializing types. I am sending the path of the type as a string and afterwards locate it to get the actual type in torch. Similar usage would be in ` serde.protobuf.native_serde.py`.

Reply:
By doing this we can drop `protobuf.torch.serde.TORCH_DTYPE_STR` in the future.